src/reactive.coffee

```diff
@@ -298,16 +290,26 @@ rxFactory = (_, $) ->
       recorder.sub (target) => rx.autoSub @onChangeCells, ([index, removed, added]) ->
         target.refresh() if removed.length != added.length
       @cells.length
-    map: (f) ->
+    map: (f) -> # TODO: Reimplement using transform for consistency?
       ys = new MappedDepArray()
       rx.autoSub @onChangeCells, ([index, removed, added]) =>
         for cell in ys.cells[index...index + removed.length]
           cell.disconnect()
-        newCells =
-          added.map (item) ->
-            cell = bind -> f(item.get())
+        newCells = added.map (item) -> bind -> f(item.get())
         ys.realSpliceCells(index, removed.length, newCells)
       ys
+    transform: (f, diff) -> new DepArray (=> f @all()), diff
```

Comment:
These all need to be deeply tested (specifying the behavior of incremental changes on the transforms)